src/dataDisplay/Tooltip/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import MaterialUiTooltip, { TooltipProps } from '@material-ui/core/Tooltip';
+import { withStyles } from '@material-ui/core';
+
+import theme from '../../theme';
+
+export type Props = Omit<TooltipProps, 'arrow'>;
+
+const StyledToolTip = withStyles(() => ({
```

Comment:
could you use `makeStyles` instead?
Heres an example https://github.com/gnosis/safe-react/blob/development/src/components/App/index.tsx#L60.